devices/debian.py

```diff
@@ -402,6 +402,41 @@ def start_sshd_server(self):
         self.sendline('/etc/init.d/ssh reload')
         self.expect(self.prompt)
 
+    def get_tftp_cfg_dest_path(self, board=None):
+        return '/tftpboot' # can this change?
+
+    def get_conf_file_from_tftp(self, board, cfg):
+        """Retrieve a file from the tftp sever"""
+        if cfg is None:
+            return None
```

Comment:
Can we make the default action to download the CM and MTA cfg files? I assume this runs on the provisioner? We should add a function there to return the current configs (as read from DHCP config), so we can just use those filenames/paths

Reply:
yes, we can, but wouldn't it be easier to get them directly from the device /var/tmp?